### ISSUE TITLE
Add exports `dist/src/index.js`

### ISSUE THREAD

**Issue by nicojs** [stryker-mutator/mutation-testing-elements#1254]
https://github.com/stryker-mutator/mutation-testing-elements/pull/1125#issuecomment-877495256

```diff
{
"main": "dist/src/index.js",
  "typings": "dist/src/index.d.ts",
  "exports": {
+    ".": "./dist/src/index.js",
    "./api": "./dist/src-generated/schema.js"
  },
}
```

@ClemensSahs 👀

**Comment by ClemensSahs** [comment]
currently merged
https://github.com/stryker-mutator/mutation-testing-elements/pull/1253

**Comment by nicojs** [comment]
That won't work. Should be a relative path.

```diff
{
"main": "dist/src/index.js",
  "typings": "dist/src/index.d.ts",
  "exports": {
 -   ".": "dist/src/index.js",
+    ".": "./dist/src/index.js",
    "./api": "./dist/src-generated/schema.js"
  },
}
```

**Comment by ClemensSahs** [comment]
thats true sorry

**Comment by nicojs** [comment]
Closed with #1255
